File: 441-arranging-coins/arranging-coins.go
func arrangeCoins(n int) int {
   rowComp:=0; 
   for n>0{
    coinUsed := rowComp +1;
    if n<coinUsed {
        break;
    }
    n = n - coinUsed;
    rowComp++;
   }
   return rowComp;
}

File: 441-arranging-coins/arranging-coins.cpp
class Solution {
public:
    int arrangeCoins(int n) {
        int rc = 0;
        while(n>0){
            int coinUsed = rc+1;
            if(n<coinUsed){
                break;
            }
            n = n - coinUsed;
            rc++;
        }
        return rc;
    }
};